### PR TITLE
Update Code Analysis overview & setup documentation 

### PR DESCRIPTION
**Pull request by kassenq**
### What does this PR do? What is the motivation?
Update Code Analysis overview & setup documentation. Motivation: Reflect updated onboarding and improve organization 

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by maycmlee** [comment]
Created [DOCS-8521](https://datadoghq.atlassian.net/browse/DOCS-8521) for editorial review.

File: content/en/code_analysis/software_composition_analysis/setup.md [hunk 1]
```diff
@@ -33,23 +33,33 @@ Code Analysis is in public beta.
 {{< /callout >}}
 
 ## Overview
+Set up Datadog Static Analysis in-app [here][1].
```
Comment (alai97) [inline content/en/code_analysis/software_composition_analysis/setup.md:36]:
```suggestion

To set up Datadog Software Composition Analysis, navigate to [**Software Delivery** > **Code Analysis**][6].
```

File: content/en/code_analysis/static_analysis/setup.md [hunk 2]
```diff
@@ -27,14 +27,34 @@ Code Analysis is in public beta.
 {{< /callout >}}
 
 ## Overview
+Set up Datadog Static Analysis in-app [here][1].
```
Comment (alai97) [inline content/en/code_analysis/static_analysis/setup.md:30]:
```suggestion
To set up Datadog Static Analysis, navigate to [**Software Delivery** > **Code Analysis**][1].
```

File: content/en/code_analysis/static_analysis/setup.md [hunk 3]
```diff
@@ -27,14 +27,34 @@ Code Analysis is in public beta.
...
+{{< whatsnext desc="See instructions based on your CI provider:">}}
+    {{< nextlink href="code_analysis/static_analysis/circleci_orbs" >}}CircleCI Orbs{{< /nextlink >}}
+    {{< nextlink href="code_analysis/static_analysis/github_actions" >}}GitHub Actions{{< /nextlink >}}
+    {{< nextlink href="code_analysis/static_analysis/generic_ci_providers" >}}Generic CI Providers{{< /nextlink >}}
+{{< /whatsnext >}}
+
+## Select your source code management provider
+Datadog Static Analysis supports all source code management providers, with native support for GitHub.
+### Set up the GitHub integration 
+If GitHub is your source code management provider, you must configure a GitHub App using the [GitHub integration tile][9] and set up the [source code integration][10] to see code snippets in-app and enable [pull request comments][11].
```
Comment (alai97) [inline content/en/code_analysis/static_analysis/setup.md:45]:
```suggestion
If GitHub is your source code management provider, you must configure a GitHub App using the [GitHub integration tile][9] and set up the [source code integration][10] to see inline code snippets and enable [pull request comments][11].
```

File: content/en/code_analysis/static_analysis/_index.md [hunk 4]
```diff
@@ -168,3 +127,5 @@ If you believe a specific violation is a false positive, you can indicate false
 [5]: /code_analysis/static_analysis/setup#add-a-static-analysis-yaml-file-to-your-project
 [6]: https://en.wikipedia.org/wiki/Camel_case
 [7]: https://en.wikipedia.org/wiki/Snake_case
+[8]: https://docs.datadoghq.com/code_analysis/static_analysis/setup/#add-a-static-analysis-yaml-file-to-your-project
```
Comment (alai97) [inline content/en/code_analysis/static_analysis/_index.md:130]:
```suggestion
[8]: /code_analysis/static_analysis/setup/#add-a-static-analysis-yaml-file-to-your-project
```

File: content/en/code_analysis/software_composition_analysis/setup.md [hunk 5]
```diff
@@ -33,23 +33,33 @@ Code Analysis is in public beta.
...
 
-To use Datadog Software Composition Analysis (SCA) in CI, you only need to set it up in its own dedicated job. 
+## Select where to run Software Composition Analysis scans
+### Scan with Datadog-hosted scanning
+SCA scans can be run directly on Datadog's infrastructure. Get started in-app [here][1].
```
Comment (alai97) [inline content/en/code_analysis/software_composition_analysis/setup.md:40]:
```suggestion
SCA scans can be run directly on Datadog's infrastructure. 
```

File: content/en/code_analysis/software_composition_analysis/setup.md [hunk 6]
```diff
@@ -33,23 +33,33 @@ Code Analysis is in public beta.
...
 
-## Enable Software Composition Analysis
+### Scan in CI pipelines
+SCA can be run in your CI pipelines using the [`datadog-ci` CLI][5]. Configure your [Datadog API and application keys][3] and run SCA jobs in the respective CI provider.
 
-Navigate to **Software Delivery** > **Code Analysis** and click **+ Add a Repository**. When setting up [Code Analysis][2] on your project, select **Enable Software Composition Analysis**.
+{{< whatsnext desc="See instructions based on your CI provider:">}}
```
Comment (alai97) [inline content/en/code_analysis/software_composition_analysis/setup.md:45]:
```suggestion
{{< whatsnext desc="See the documentation for your CI provider:">}}
```

File: content/en/code_analysis/software_composition_analysis/setup.md [hunk 7]
```diff
@@ -33,23 +33,33 @@ Code Analysis is in public beta.
...
+    {{< nextlink href="code_analysis/software_composition_analysis/github_actions" >}}GitHub Actions{{< /nextlink >}}
+    {{< nextlink href="code_analysis/software_composition_analysis/generic_ci_providers" >}}Generic CI Providers{{< /nextlink >}}
+{{< /whatsnext >}}
 
-{{< img src="code_analysis/software_composition_analysis/enable_sca.png" alt="Click the Enable Software Composition Analysis checkbox on the Code Analysis setup page when setting up Code Analysis for your project" style="width:100%;" >}}
+## Select your source code management provider
+Datadog SCA supports all source code management providers, with native support for GitHub.
+### Set up the GitHub integration 
+If GitHub is your source code management provider, you must configure a GitHub App using the [GitHub integration tile][9] and set up the [source code integration][10] to see code snippets in-app and enable [pull request comments][11].
```
Comment (alai97) [inline content/en/code_analysis/software_composition_analysis/setup.md:53]:
```suggestion
If GitHub is your source code management provider, you must configure a GitHub App using the [GitHub integration tile][9] and set up the [source code integration][10] to see inline code snippets and enable [pull request comments][11].
```

File: content/en/code_analysis/software_composition_analysis/setup.md [hunk 8]
```diff
@@ -33,23 +33,33 @@ Code Analysis is in public beta.
...
+- `Content: Read`, which allows you to see code snippets displayed in Datadog
+- `Pull Request: Read & Write`, which allows Datadog to add feedback for violations directly in your pull requests using [pull request comments][11]
 
-{{< whatsnext desc="See the documentation for information about the following integrations:">}}
-    {{< nextlink href="code_analysis/software_composition_analysis/github_actions" >}}GitHub Actions{{< /nextlink >}}
-    {{< nextlink href="code_analysis/software_composition_analysis/generic_ci_providers" >}}Generic CI Providers{{< /nextlink >}}
-{{< /whatsnext >}}
+### Other source code management providers
+If you are using another source code management provider, configure SCA to run in your CI pipelines via the datadog-ci CLI tool and [upload the results][8] to Datadog.
+You **must** run an analysis of your repository on the default branch before results can begin appearing in the Code Analysis page.
```
Comment (alai97) [inline content/en/code_analysis/software_composition_analysis/setup.md:62]:
```suggestion
You **must** run an analysis of your repository on the default branch before results can begin appearing on the **Code Analysis** page.
```

File: content/en/code_analysis/static_analysis/_index.md [hunk 9]
```diff
@@ -45,15 +45,17 @@ Static Analysis supports scanning for poor coding practices and security vulnera
 
 {{< partial name="code_analysis/languages-getting-started.html" >}}
 
+Set up Datadog Static Analysis in-app [here][1], or follow [setup documentation here][9].
+
 </br>
```
Comment (alai97) [inline content/en/code_analysis/static_analysis/_index.md:50]:
```suggestion
</br> 

To get started, you can set up Static Analysis on the [**Code Analysis** page][1] or see the [Setup documentation][9].
```

**Review by alai97 — COMMENTED** [review]
Looks good, some copy suggestions!

**Review by alai97 — APPROVED** [review]
LGTM

**Pull request merged**